examples/graphql-mutations/graphql_mutations.bal

```diff
@@ -0,0 +1,63 @@
+import ballerina/graphql;
+
+service /graphql on new graphql:Listener(4000) {
+
+    // Define a `Person` object in the service
+    private Person person;
+
+    function init() {
+        // Initialize the `person` value
+        self.person = new("Walter White", 51);
+
```

Comment:
Do we need above new lines ?

Reply:
To show how we serially change the object, I felt like this is the easiest way. We keep a class-level variable, and we mutate it in two steps. That's why used defined a class variable.